src/components/AccountTransactionsTable.vue

```diff
@@ -60,6 +68,7 @@ import DatetimeLabel from '@/components/DatetimeLabel'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import { accountHints } from '@/utils/hints/accountHints'
 import HintTooltip from '@/components/HintTooltip'
+import { transactionsHints } from '~/utils/hints/transactionsHints'
```

Comment:
I would change it to:

```suggestion
import { transactionsHints } from '@/utils/hints/transactionsHints'
```

Reply:
right, fixed

Reply:
It was present on more places, also fixed